lua/.gitignore

```diff
@@ -0,0 +1,3 @@
+/rootfs/
+/workdir/
+initrd.cpio
```

Comment:
```suggestion
```

Not required. It's ignored by the [top-level `.gitignore` file](https://github.com/unikraft/catalog-core/blob/main/.gitignore).